crates/exchange-rates/src/init.rs

```diff
@@ -0,0 +1,47 @@
+use async_trait::async_trait;
+use iron_types::GlobalState;
+use once_cell::sync::OnceCell;
+use serde::Deserialize;
+use std::collections::HashMap;
+use tokio::sync::{RwLock, RwLockReadGuard, RwLockWriteGuard};
+
+use super::feed::Feed;
+use super::Feeds;
+
+static FEEDS: OnceCell<RwLock<Feeds>> = OnceCell::new();
```

Comment:
instead of `OnceCell` and an `init` function, this can be simplified with `Lazy`

static FEEDS: Lazy<RwLock<Feeds>> = Lazy::new(|| {
  let feeds: Feeds = ...;

  RwLock::new(feeds)
});
```

the custom init function is only needed when we need additional arguments